Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2234,6 +2236,11 @@ def __init__(self, id, cvss, published, modified, description):
             self.published = published
             self.modified = modified
             self.description = description
+            self.dbot_score = None
+
+        def set_dbot_score(self, dbot_score):
```

Comment:
no need for setter. because there is no meaning for DBotScore for CVE, and it will be always 0, you can create DBotScore object in the ctor yourself.